STARTER/STARTER.theme

```
@@ -1,9 +1,57 @@
 <?php
 
 /**
- * Include all files from the includes directory
+ * @file
+ * STARTER Drupal 8 theme based on Draft theme.
  */
-$includes_path = dirname(__FILE__) . '/includes/*.inc';
-foreach (glob($includes_path) as $filename) {
-  require_once dirname(__FILE__) . '/includes/' . basename($filename);
+
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Utility function to get the inline css style for a background image.
+ *
+ * @param mixed $entity
+ *   Entity containing the image field.
+ * @param string $field_name
+ *   Name of the image field to use.
+ * @param string $image_style
+ *   Name of the image style to use.
+ *
+ * @return string
+ *   Full css style attribute containing background image.
+ */
+function _get_entity_image_inline_style($entity, $field_name, $image_style) {
+  $rendered_style = '';
+
+  if (!$entity->$field_name->isEmpty()) {
```

We might want to raise `InvalidArgumentException` when no field found. Let's think about it